index.rst

```diff
@@ -123,6 +123,7 @@ Sensor Components
     INA219, components/sensor/ina219, ina219.jpg
     INA226, components/sensor/ina226, ina226.jpg
     INA3221, components/sensor/ina3221, ina3221.jpg
+    Inkbird IBS-TH1 Mini, components/sensor/inkbird_ibsth1_mini, inkbird_isbth1_mini.png
```

Comment:
This picture needs to be minified further (148kb is a lot for the main page). I'll update it in a second